test/rendering/TestApp.jsx

```diff
@@ -63,6 +63,7 @@ class TestApp extends React.Component {
       enableForwardSkip: true,
       ...getP('startingQuestionId', 'layout-options-65'),
       startingStep: 1,
+        debugMode: true,
```

Comment:
We want to have typical usage of the form in TestApp and thus we want:
```suggestion
        debugMode: false,
```